api/main.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import FastAPI, File, UploadFile
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"This is": "a test"}
+
+
+@app.post("/upload/")
+async def upload(file: UploadFile):
+    contents = await file.read()  # --> file is a standard python file object
+    # TODO: Add a function call here to the ML model which will accept the file object
+
+    # temporary code to write image on to the disk
+    with open("tmp/img.jpeg", "wb") as f:
```

Comment:
Even though this is temporary code, this line of code would throw an error on a computer which doesn't have a tmp file. You could write a fail safe by creating the folder if it doesn't exist.

Reply:
Ok I will make the changes

Reply:
Changes made